styles.css

```diff
@@ -139,3 +161,175 @@ h4 {
   margin: 24px auto 62px;
   background-color: var(--secondary);
 }
+
+.Work-projects {
+  display: grid;
+  justify-content: center;
+  padding: 16px;
+  gap: 12px;
+  width: 100%;
+  height: auto;
+}
+
+.card {
+  background-color: white;
+  padding: 2%;
+  border-radius: 15px;
+  margin: 5% 5% 5% 5%;
+  padding-left: 7%;
+}
+
+.projects-done h2 {
+  display: flex;
+  justify-content: flex-start;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.projects-done .time {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+}
+
+.projects-done ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  margin-right: 40%;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+h3 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills {
+  display: flex;
+  justify-content: space-around;
+  font-family: Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  padding: 10px;
+  gap: 5px;
+}
+
+.skill {
+  position: relative;
+  background: #ebebff;
+  border-radius: 6px;
+  padding: 4px;
+  list-style-type: none;
+}
+
+.skill-content {
+  position: relative;
+  color: black;
+}
+
+.button {
```

Comment:
- [ ] The section has the margins (top, bottom, left, right) defined in the design , kindly use them as mentioned on figma design 😉